src/lib/components/quiz/quiz.svelte

```diff
@@ -0,0 +1,249 @@
+<script lang="ts">
+  import type { QuizData, TeamMatch } from '$lib/constants/quiz';
+  import LeftArrow from '$lib/components/icons/left-arrow.svelte';
+  import RightArrow from '$lib/components/icons/right-arrow.svelte';
+  import MoreInfo from './more-info.svelte';
+
+  export let data: QuizData;
+
+  $: index = 0;
+  let response: TeamMatch[] = [];
+  $: answeredAllQuestions =
+    response.length === data.questions.length && !response.includes(undefined);
+  let showResults = false;
+  let showMoreInfo = false;
+  let showTeam: string;
+  $: talliedResponses = response.reduce((tallies, match) => {
+    if (tallies[match]) tallies[match]++;
+    else tallies[match] = 1;
+    return tallies;
+  }, {});
+  let teams = ['Algo', 'Dev', 'Design', 'AI'];
+  let match: string;
```

Comment:
We can calculate the resulting team match reactively, using `talliedResponses` as a dependency.
```suggestion
  $: match = Object.entries(talliedResponses).sort(([,a], [,b]) => b - a).shift().shift();
```

Reply:
done